indra/llrender/llrender.cpp

```diff
@@ -1580,7 +1580,12 @@ void LLRender::flush()
 	if (mCount > 0)
 	{
         LL_PROFILE_ZONE_SCOPED_CATEGORY_PIPELINE;
-        llassert(LLGLSLShader::sCurBoundShaderPtr != nullptr);
+        if (!LLGLSLShader::sCurBoundShaderPtr)
+        {
+            // The shader was not created yet? Report and shatdown the application
+            LL_ERRS() << "There are graphical problems (no shader), try updating driver" << LL_ENDL;
+        }
+
```

Comment:
`LLRender::flush` is called in a lot of different places. It would make more sense to check for failed shader compilation/linking.